dces-drc-integration/src/test/java/uk/gov/justice/laa/crime/dces/integration/utils/ServiceSchedulerTest.java

```diff
@@ -34,16 +35,19 @@
         "spring.main.allow-bean-definition-overriding=true",
         "scheduling.cron.process-fdc-files=* * * * * *",
         "scheduling.cron.process-contributions-files=* * * * * *",
+        "scheduling.cron.data-migration=* * * * * *",
```

Comment:
Question, not a requested change: As you're not testing the data-migration scheduling here, would it make more sense to disable it (`-` instead of `* * * * * *`), so the test is testing the normal way the integration runs?